addons/website_slides/static/src/js/portal_rating_composer.js

```diff
@@ -5,6 +5,6 @@ RatingPopupComposer.include({
         this._super(...arguments);
         this.options.force_submit_url =
             data.force_submit_url ||
-            (this.options.default_message_id && "/slides/mail/update_comment");
+            (this.options.default_message_id && "/mail/message/update_content");
```

Comment:
What's the point of `force_submit_url`? Do we ever give it another URL? And if so which one?

Reply:
Yes, posting a message and editing a message both go through the same process in rating composer widget. The other one is `/mail/message/post`.

Note: `force_submit_url` will be deleted when the current widget is replaced by the `composer` component.